core/src/main/java/org/n52/iceland/ogc/gml/time/TimePosition.java

```diff
@@ -62,8 +54,7 @@ public TimePosition(DateTime time) {
      *            Indeterminate value of time position
      */
     public TimePosition(TimeIndeterminateValue indeterminateValue) {
-        super();
-        this.indeterminateValue = indeterminateValue;
+        this.indeterminateValue = Optional.of(indeterminateValue);
```

Comment:
I'd vote `Time` instances for being immutable. Once created why should it change? In contrast to a `Car` instance which do have mutable properties, e.g. `fuel`, `speed`. But a `Time` instant naturally is immuntable IMHO.